pages/community/foundation/index.tsx

```diff
@@ -224,8 +223,6 @@ export default function Governance() {
           }}
         />
       </Box>
-
```

Comment:
Slightly unrelated, but maybe the "Grants" section on this page should link to the grants page and say "Learn more" instead of "Apply Now"?

Reply:
I'm gonna merge this as is but I'll ask about making this change cause I agree that would be better.